server/src-lib/Hasura/GraphQL/Execute/LiveQuery/Plan.hs

```diff
@@ -281,24 +348,29 @@ reuseLiveQueryPlan
   -> Maybe GH.VariableValues
   -> ReusableLiveQueryPlan
   -> m LiveQueryPlan
-reuseLiveQueryPlan pgExecCtx sessionVars queryVars reusablePlan = do
-  let ReusableLiveQueryPlan parameterizedPlan syntheticVars queryVarTypes = reusablePlan
-  annVarVals <- GV.validateVariablesForReuse queryVarTypes queryVars
+reuseLiveQueryPlan pgExecCtx sessionVars queryVars ReusableLiveQueryPlan{..} = do
+  annVarVals <- GV.validateVariablesForReuse _rlqpQueryVariableTypes queryVars
   validatedVars <- validateVariables pgExecCtx annVarVals
-  pure $ LiveQueryPlan parameterizedPlan (CohortVariables sessionVars validatedVars syntheticVars)
+  pure $ LiveQueryPlan _rlqpParameterizedPlan $
+         mkCohortVariables _rlqpRequiredSessionVariables
+         sessionVars validatedVars _rlqpSyntheticVariableValues
 
 data LiveQueryPlanExplanation
   = LiveQueryPlanExplanation
-  { _lqpeSql  :: !Text
-  , _lqpePlan :: ![Text]
+  { _lqpeSql       :: !Text
+  , _lqpePlan      :: ![Text]
+  , _lqpeVariables :: !CohortVariables
```

Comment:
I'm wondering if we should also include the parametrized sql query here.

Reply:
Isn’t that what `_lqpeSql` is?